config.json

```diff
@@ -1348,6 +1348,18 @@
         "math"
       ]
     },
+    {
+      "slug": "knapsack",
+      "uuid": "ac179b77-98a5-4daf-9773-3d68b6cd8548",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 8,
```

Comment:
I would actually say it is more like a difficulty 4-5 problem.

Reply:
Updated